src/handlers/transfers/FxFulfilService.js

```diff
@@ -78,23 +87,39 @@ class FxFulfilService {
   async validateHeaders({ transfer, headers, payload }) {
     let fspiopError = null
 
-    if (headers[SOURCE]?.toLowerCase() !== transfer.counterPartyFspName.toLowerCase()) {
+    if (
+      headers[SOURCE]?.toLowerCase() !==
+      transfer.counterPartyFspName.toLowerCase()
+    ) {
       fspiopError = fspiopErrorFactory.fxHeaderSourceValidationError()
     }
-    if (headers[DESTINATION]?.toLowerCase() !== transfer.initiatingFspName.toLowerCase()) {
+    if (
+      headers[DESTINATION]?.toLowerCase() !==
+      transfer.initiatingFspName.toLowerCase()
+    ) {
       fspiopError = fspiopErrorFactory.fxHeaderDestinationValidationError()
     }
 
     if (fspiopError) {
-      const apiFSPIOPError = fspiopError.toApiErrorObject(this.Config.ERROR_HANDLING)
+      const apiFSPIOPError = fspiopError.toApiErrorObject(
+        this.Config.ERROR_HANDLING
+      )
       const eventDetail = {
         functionality: Type.POSITION,
         action: Action.FX_ABORT_VALIDATION
       }
-      this.log.warn('headers validation error', { eventDetail, apiFSPIOPError })
+      this.log.warn('headers validation error', {
```

Comment:
What is the point of such reformatting? I'd like to have log only on one line, not 4

Reply:
@geka-evk Could be the linter in the pre-commit hook. Will fix that.